data/json/recipes/ammo/shot.json

```diff
@@ -613,5 +613,23 @@
     "proficiencies": [ { "proficiency": "prof_handloading" } ],
     "tools": [ [ [ "press", -1 ] ] ],
     "components": [ [ [ "incendiary", 25 ] ] ]
+  },
+  {
+    "result": "wax_slug",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_SHOT",
+    "skill_used": "fabrication",
+    "difficulty": 3,
```

Comment:
This seems like a pretty high fabrication level for what is melting wax. Maybe make it require 1 level of shotguns to simulate someone being more familiar with how shot works, and then figuring out you can add wax?